common/aggregate_functions/src/aggregate_arg_max.rs

```diff
@@ -59,6 +59,9 @@ impl AggregateFunction for AggregateArgMaxFunction {
     }
 
     fn accumulate(&mut self, columns: &[DataColumnarValue], _input_rows: usize) -> Result<()> {
+        if _input_rows == 0 {
```

Comment:
_input_rows -> input_rows maybe is better.

Reply:
Done